config/static/single-pod/pod-main.yaml

```diff
@@ -144,6 +144,12 @@ spec:
       privileged: false
     terminationMessagePath: /dev/termination-log
     terminationMessagePolicy: File
+    # https://kubernetes.io/docs/tasks/configure-pod-container/attach-handler-lifecycle-event/
+    # https://www.freedesktop.org/software/systemd/man/systemd.html#SIGRTMIN+3
```

Comment:
Thank you for putting references in the comments :ok_hand:

Reply:
you are welcome! :)